### PR TITLE
Readds Baneposting

### PR DESCRIPTION
**Pull request by ArcLumin**
This has been gone for FAR too long. I've tweaked it a bit;

Emagging the sec hailer will now make it play only baneposts

From https://github.com/HippieStationCode/HippieStation13/pull/1616

### CODE REVIEW

**Comment by trifornt-tinfornt** [comment]
Emagging it should be the only way to make it banepost.

**Comment by ArcLumin** [comment]
Before it was using wirecutters on it would let it banepost, so I kept it in

File: code/modules/clothing/masks/hailer.dm [hunk 1]
```diff
@@ -108,7 +114,9 @@
 			if(3)
 				phrase = rand(1,18)	// user has unlocked all phrases, set upper limit to last phrase. The mask will play all phrases
 			if(4)
-				phrase = rand(12,18)	// user has broke the restrictor, it will now only play shitcurity phrases
+				phrase = rand(12,26)	// user has broke the restrictor, it will now only play shitcurity phrases
```
Comment (ghost) [inline code/modules/clothing/masks/hailer.dm:118]: Shouldn't the comment be shitcurity AND baneposts

Comment (ArcLumin) [reply code/modules/clothing/masks/hailer.dm:118]: I mean
Doesn't baneposting by definition make you shitcurity?

**Comment by doobiedoo23** [comment]
thanks for the credit goy

**Comment by ArcLumin** [comment]
You're welcome lad

**Comment by ghost** [comment]
If I approve this PR, will you die?

**Comment by TDcoolguy300** [comment]
Very painfully

**Comment by ArcLumin** [comment]
For you

**Comment by ghost** [comment]
FUCK I SHOULD HAVE SAID "IF I PULL THIS IN, WILL YOU DIE?"

**Comment by ghost** [comment]
But +1 very nice

**Comment by ghost** [comment]
the fire rises
+1/3

**Comment by Hackole** [comment]
yes please +1/3

**Comment by Kokojos** [comment]
Meh +1

**Pull request merged**